shared/actions/tracker.js

```diff
@@ -51,14 +51,13 @@ export function stopTimer (): Action {
 }
 
 export function registerTrackerChangeListener (): TrackerActionCreator {
-  return dispatch => {
+  return (dispatch, getState) => {
     const params = {
       'keybase.1.NotifyTracking.trackingChanged': ({username}) => {
-        dispatch({
-          type: Constants.userUpdated,
-          payload: {},
-        })
-        dispatch(getProfile(username))
+        const trackerState = getState().tracker.trackers[username]
+        if (trackerState && trackerState.type === 'tracker' && !trackerState.closed) {
```

Comment:
I meant that when you navigate to a search/profile it calls getProfile a different way.

There is only one way we update from the service, which is this code path. I changed it so it will update if there is a trackerState at all (not just one that has !.closed)